drivers/bme280/Makefile

```diff
@@ -1,3 +1,5 @@
 MODULE = bme280
 
+DIRS = $(RIOTBASE)/drivers/bmx280-common
```

Comment:
I'm not sure about this line... maybe it should be `DIRS +=` ?

Since it seems to be the first "shared code" driver is difficult to say what's the preferred method to add the common code folder...